lib/LTP/WhiteList.pm

```diff
@@ -36,7 +36,9 @@ sub _download_whitelist {
     my $path = get_var('LTP_KNOWN_ISSUES');
     return undef unless defined($path);
 
-    my $res = Mojo::UserAgent->new->get($path)->result;
+    my $ua = Mojo::UserAgent->new;
+    $ua->max_redirects(5);
```

Comment:
Looks good, just note that the following should work as well:
```suggestion
    my $ua = Mojo::UserAgent->new(max_redirects => 5);
```

Reply:
Let's use the shorter variant then. I've also added another commit which make the warning message clearer.